plugins/advisors/vulnerable-code/src/funTest/kotlin/VulnerableCodeFunTest.kt

```diff
@@ -30,27 +30,51 @@ import org.ossreviewtoolkit.model.Package
 import org.ossreviewtoolkit.model.utils.toPurl
 import org.ossreviewtoolkit.utils.test.shouldNotBeNull
 
-class VulnerableCodeFunTest : StringSpec({
-    "Findings should be returned for a vulnerable package" {
-        val vc = VulnerableCode("VulnerableCode", VulnerableCodeConfiguration())
-        val id = Identifier("Maven:com.google.guava:guava:19.0")
-        val pkg = Package.EMPTY.copy(id, purl = id.toPurl())
-
-        val findings = vc.retrievePackageFindings(setOf(pkg))
-
-        findings.values.flatMap { it.summary.issues } should beEmpty()
-        with(findings.values.flatMap { it.vulnerabilities }.associateBy { it.id }) {
-            keys shouldContainAll setOf(
-                "CVE-2018-10237",
-                "CVE-2020-8908",
-                "CVE-2023-2976"
-            )
-
-            getValue("CVE-2023-2976").references.find {
-                it.url.toString() == "https://nvd.nist.gov/vuln/detail/CVE-2023-2976"
-            } shouldNotBeNull {
-                severity shouldBe "7.1"
-                severityRating shouldBe "HIGH"
+class VulnerableCodeFunTest : WordSpec({
+    "Vulnerable Maven packages" should {
+        "return findings for Guava" {
+            val vc = VulnerableCode("VulnerableCode", VulnerableCodeConfiguration())
+            val id = Identifier("Maven:com.google.guava:guava:19.0")
+            val pkg = Package.EMPTY.copy(id, purl = id.toPurl())
+
+            val findings = vc.retrievePackageFindings(setOf(pkg))
```

Comment:
commit: Why do we need a test for another Maven package?

Reply:
Please see [my comment above](https://github.com/oss-review-toolkit/ort/pull/8256#issuecomment-1938161046).